docs/resources/logdna_alert.md

```diff
@@ -0,0 +1,117 @@
+# logdna_alert Resource
+
+Manages [LogDNA alert presets](https://docs.logdna.com/docs/alerts). To get started, all you need to do is to specify a `name` and configuration for one of our currently supported alerts- email, webhook, or PagerDuty.
```

Comment:
`[LogDNA Alert Presets]` instead of `[LogDNA alert presets]` would be more appropriate, I think